cmd/cli/cli.go

```diff
@@ -93,7 +95,9 @@ func RunCli(supportedSources []config.ConfigurationPair, supportedTransformation
 		}
 	}
 
-	app.Run(os.Args)
+	if err := app.Run(os.Args); err != nil {
+		log.WithError(err).Error("failed to run cli")
```

Comment:
```suggestion
		log.Fatalf("failed to run cli: %s", err.Error())
```

If this happens, we should crash immediately (this was a flaw in the previous implementation, which I'm quite happy to see surfaced. :)

Reply:
`WithError` would add the error message as an attribute to the log entry, which feels nicer (especially if you feed those logs into Grafana etc); unless we want it to become a part of the error message string ofc 👍

Reply:
combined my approach with your suggestion in [7fe4243](https://github.com/snowplow/snowbridge/pull/412/commits/7fe42433fd6e0992956e9e6fae93e19a3ce8ec5b)

Reply:
Yeah that works, I just meant to illustrate that we should crash